### DISCUSSION TITLE
Fixed Header remains at top of the page

### DISCUSSION THREAD

**Discussion by sergey-dme** [Budibase/budibase#1752] [Ideas]
It would be great if the Header w/menu will be a fixed - always remains at the top of the page as user scrolls up and down - like on most websites.

**Comment by shogunpurple** [comment]
@sergey-dme you’ll be glad to know this one will be in the release next week too!

**Comment by sergey-dme** [comment]
@shogunpurple oh wow! That's cool. Great news! Very excited 😊
